packages/terra-alert/docs/README.md

```diff
@@ -7,7 +7,6 @@ The Terra Alert component is a notification banner that can be rendered in your
 
 - Install with [npmjs][3]:
   - `npm install terra-alert`
-  - `yarn add terra-alert`
```

Comment:
Are these auto-generated when creating a new package?

Reply:
At one point, there were auto-generated, but we've made changes to remove this for new packages: https://github.com/cerner/generator-terra-module/pull/19/files#diff-4b8982f7ad1a52f85e37add894537d2fL20